src/commands/positive.ts

```diff
@@ -192,18 +198,37 @@ export default class Positive extends Command {
     const securityTypes = {};
     for (const scheme of securitySchemes) {
       if (this.securities.includes(scheme.key)) {
-        securityTypes[scheme.type] = scheme.key;
+        securityTypes[scheme.type] = {
+          type: scheme.type,
+          key: scheme.key,
+          scheme: scheme.scheme,
+        };
       }
     }
 
     if (securityTypes[OASSecurityType.APIKEY]) {
-      const apiSecurityName = securityTypes[OASSecurityType.APIKEY];
+      const apiSecurityName = securityTypes[OASSecurityType.APIKEY].key;
       const value =
         flags.apiKey ??
-        (await cli.prompt('What is your API Key?', { type: 'mask' }));
+        (await cli.prompt('Please provide your API Key?', { type: 'mask' }));
 
       this.securityValues[apiSecurityName] = { value };
     }
+
+    if (securityTypes[OASSecurityType.HTTP]) {
+      const bearerSecurityName = securityTypes[OASSecurityType.HTTP].key;
+      let value;
+
+      if (securityTypes[OASSecurityType.HTTP].scheme === 'bearer') {
```

Comment:
Might be nice to make 'bearer' a constant. We have one other defined in the `src/utilities/constants.ts` file

Reply:
Great suggestion!